libvisual-plugins/plugins/actor/gstreamer/actor_gstreamer.c

```diff
@@ -90,26 +91,41 @@ static int act_gstreamer_init (VisPluginData *plugin)
 
     gst_init (NULL, NULL);
 
-    char *launch_str = g_strdup_printf ("filesrc location=%s ! decodebin ! ffmpegcolorspace ! "
+    // Regarding the pipeline below:
+    // - element "filesrc" reads a video file from disk
+    // - element "decodebin" detects the video file's encoding
+    //   and serves raw pixel data
```

Comment:
Let's use the description from `decodebin`'s [documentation](https://gstreamer.freedesktop.org/documentation/playback/decodebin.html?gi-language=c). It is more accurate and informative. 

_element "decodebin" auto-magically constructs a decoding pipeline using available decoders and demuxers via auto-plugging._

Reply:
@kaixiong I understand why you consider that text more suitable. Alright, pushed.